### PR TITLE
Hook up linker to nodejs_binary/nodejs_test

### PR DESCRIPTION
**Pull request by alexeagle**
For #1382 
Pairing with @mrmeku on it today

### CODE REVIEW

File: examples/jest/package.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
     },
     "scripts": {
         "test": "bazel test ...",
-        "postinstall": "patch-package --patch-dir=./tools/patches"
+        "// postinstall": "patch-package --patch-dir=./tools/patches"
```
Comment (gregmagolan) [inline examples/jest/package.json]: remove entirely?

**Comment by mrmeku** [comment]
@alexeagle Looks like the only failing test was already failing in prior builds: https://buildkite.com/bazel/rules-nodejs-nodejs/builds/5046

**Comment by gregmagolan** [comment]
Ahh. Thanks for pointing that out. It's a strange heisenbug that shows up only on bazelci osx. I haven't been able to reproduce it locally either.

**Comment by mrmeku** [comment]
@gregmagolan The package the error references is `jest-haste-map` so maybe try to lock down the dependency for that dependency even more by using the `resolutions` section of package.json: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

And if that doesn't work just pray to your chosen deity and ask for forgiveness

**Pull request merged**